### DISCUSSION TITLE
Add lazer for download to Linux Repositories

### DISCUSSION THREAD

**Discussion by treborrobertgit** [ppy/osu#18346] [Ideas] [answered]
Will lazer be added to Linux repos like Snapcraft for download? This would make installation and updates easier.

**Comment by integerrr** [comment, marked as answer]
#2960
#3727
#7840
https://github.com/ppy/osu/issues/3471#issuecomment-423773506
#15278
#18274
tl;dr no the maintainers won't add it themselves but the repositories for those distributions are free to take the appimage and extract for users

**Reply by smoogipoo** [reply]
I wouldn’t say “no”, just the maintainers will not be adding it. Repositories are free to take the .AppImage we provide and extract/“install” it for users for whatever distribution. Flathub already does this.

**Reply by integerrr** [reply]
Yeah sorry that's what I was getting at but failed to reflect in the original comment. Edited comment to clarify.
